language/move-model/src/model.rs

```diff
@@ -2933,7 +2934,7 @@ pub struct FunctionData {
 
     /// List of type argument names. Not in bytecode but obtained from AST.
     #[allow(unused)]
-    type_arg_names: Vec<Symbol>,
+    type_arg_names: Vec<(Symbol, Type)>,
```

Comment:
Maybe this field should be renamed since it's no longer just names of the type args?

Reply:
Good call! Done!